app/javascript/dashboard/routes/dashboard/settings/integrations/Webhooks/WebhookForm.vue

```diff
@@ -111,6 +140,29 @@ export default {
           :placeholder="webhookNameInputPlaceholder"
         />
       </label>
+      <label>
+        {{ $t('INTEGRATION_SETTINGS.WEBHOOK.FORM.INBOX.LABEL') }}
+        <!-- NOTE: 16px bottom margin to match default input margin for even spacing -->
+        <MultiselectDropdown
+          class="mb-4"
+          :options="inboxesList"
+          :selected-item="assignedInbox"
+          :multiselector-title="
+            $t('INTEGRATION_SETTINGS.WEBHOOK.FORM.INBOX.TITLE')
+          "
+          :multiselector-placeholder="
+            $t('INTEGRATION_SETTINGS.WEBHOOK.FORM.INBOX.PLACEHOLDER')
+          "
+          :no-search-result="
+            $t('INTEGRATION_SETTINGS.WEBHOOK.FORM.INBOX.NO_RESULTS')
+          "
+          :input-placeholder="
+            $t('INTEGRATION_SETTINGS.WEBHOOK.FORM.INBOX.INPUT_PLACEHOLDER')
+          "
+          :disabled="isEditing"
```

Comment:
As mentioned in previous comment, we might want to remove the "disabled on edit" feature.